playbook-website/app/views/samples/show.html.erb

```diff
@@ -39,7 +39,7 @@
           <%= pb_rails("nav", props: { orientation: "horizontal", variant: "subtle" }) do %>
             <%= pb_rails("nav/item", props: { text: "Rails", link: "#{sample_show_path(@sample, type: :rails)}", active: current_sample_type.rails? }) %>
             <%= pb_rails("nav/item", props: { text: "React", link: "#{sample_show_path(@sample, type: :react)}", active: current_sample_type.react? }) %>
-            <%= pb_rails("nav/item", props: { text: "Swift", link: "#{sample_show_path(@sample, type: :swift)}", active: current_sample_type.swift? }) if Rails.env.development? %>
+            <%= pb_rails("nav/item", props: { text: "Swift", link: "#{sample_show_path(@sample, type: :swift)}", active: current_sample_type.swift? }) %>
```

Comment:
We don't have any swift samples. This causes a 404 when clicked. Can we just remove this whole line? I'm not sure why swift samples were created as we have no way of doing them.

Reply:
Removing the line seems to be the best option, not sure when and why this was added for samples